### ISSUE TITLE
RHC - Lord Ahune Emblem Issue

### ISSUE THREAD

**Issue by GustafGSvensson** [SallyWhitemane/Mograine#1312]
**Description:**

The Frost Lord Ahune dungeon doesn't give you the Satchel of Chilled Goods if you do the random heroic daily first, and if you do The Frost Lord Ahune you can't get the triumphs from the random heroic daily, that shouldn't be the case, they should be separate dailies from each other. Also the Satchel of Chilled Goods should give current emblems but they are giving emblems from the previous patch instead. Essentially running the event dungeon first locks you out of 2 triumphs for that day.

**How to reproduce:**

Do Lord Ahune first and then try to complete RHC.

**How it should work:**

It should give you EOT + Satchel - allowing you to get satchel from Ahune or have a seperate queue from the RHC and just give a satchel.

**Comment by Veresa** [comment]
I agree with the fact that that event should give bag+Triumph if done first and once done it is impossible to get them even from that hero for not having double reward.
Then DF event both bag and Triumph or DF hero only Triumph.
